kubectl-minio/main.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020, MinIO, Inc.
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package main
+
+import (
+	"os"
+
+	"github.com/minio/kubectl-minio/cmd"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var version = "DEVELOPMENT.GOGET"
```

Comment:
We don't need this just set it to `0.0.0` , we shall use semver here and even match this to operator release tags for now.